VideoLocker/src/main/java/org/edx/mobile/view/adapters/TopicSpinnerAdapter.java

```diff
@@ -0,0 +1,4 @@
+package org.edx.mobile.view.adapters;
+
+public class TopicSpinnerAdapter {
+}
```

Comment:
Yeah :wink: I meant to move the adapter logic out of DiscussionAddPostFragment to keep it clean. Actually did this now.